pages/getData.tsx

```diff
@@ -2,14 +2,88 @@ import type { NextPage } from 'next';
 import Head from 'next/head';
 import { useState } from 'react';
 import { isAddress } from 'web3-utils';
+import {
+  FormControl,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+} from '@mui/material';
```

Comment:
I'm trying to get rid of Mui - can you use this instead: https://bulma.io/documentation/form/select/ ?